pom.xml

```diff
@@ -1811,14 +1811,18 @@ under the License.
 				<plugin>
 					<groupId>pl.project13.maven</groupId>
 					<artifactId>git-commit-id-plugin</artifactId>
-					<!-- Don't use 2.1.14 as it is incompatible with various maven versions -->
-					<version>2.1.10</version>
+					<version>4.0.0</version>
```

Comment:
why are we changing the version?

Reply:
I generally update dependencies when the result still works.
In this case there were also a few CVEs that were fixed and more compatibility in building in the various clouds: https://github.com/git-commit-id/git-commit-id-maven-plugin/issues?q=milestone%3A4.0.0